terraform/vars/terraform-dev.tfvars

```diff
@@ -4,4 +4,4 @@ service_url               = ""
 desired_count             = 1
 auto_scaling_min_capacity = 1
 auto_scaling_max_capacity = 5
-lambda_analysis_workspace = "feature-FanOutFanIn"
\ No newline at end of file
+lambda_analysis_workspace = "features-lat_lon"
```

Comment:
No need to act right now, but we should look into otherwise to link to the right lambda function in dev

- integrate lambda into api repo?
- always push to production and build against distinct deployment versions?
- other ideas?